mongodump/mongodump.go

```diff
@@ -38,22 +38,24 @@ type MongoDump struct {
 	InputOptions  *InputOptions
 	OutputOptions *OutputOptions
 
+	SkipUsersAndRoles bool
```

Comment:
Yeah, I got that. But for someone coming in to learn the codebase or make a change, this is one of the questions they are going to ask, guaranteed.